docs/getting-started-with-scalardb-on-cosmosdb.md

```diff
@@ -10,8 +10,8 @@ ScalarDB is written in Java. So the following software is required to run it.
 * [Oracle JDK 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) (or OpenJDK 8)
 * Other libraries used from the above are automatically installed through gradle
 
-## Cosmos DB setup
-You also need to set up a Cosmos DB account to get started with ScalarDB on Cosmos DB.
+## Cosmos DB for NoSQL setup
+You also need to set up a Cosmos DB for NoSQL account to get started with ScalarDB on Cosmos DB for NoSQL.
 
 This section explains how to set up [Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/introduction) with Azure portal.
 1. Select **Azure Cosmos DB** service from the services on Azure portal.
```

Comment:
I noticed that these steps are a bit different from the current steps.
I think it is better to add official document links to reduce the maintenance cost of this document.
So, I will update these steps in another PR.